src/summary/components/TaskView.test.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * @vitest-environment jsdom
+ */
+
+import { expect, test } from "vitest"
+import { render, screen } from "test/utils"
+import { TaskView } from "./UtilsViews"
+
+test("renders Only Utils/Task view ", async () => {
+  const createdAtStr: string = "2024-03-27 11:43 PM"
+  const task1CreatedAt: Date = new Date(createdAtStr)
+  const exptask1CreatedAt = "March 27, 2024 at 23:43:00"
+  const userLastName = "user lastname"
+  const userFirstName = "user firstname"
+  const task1 = {
+    name: "task1",
+    description: "a task test ",
+    createdAt: task1CreatedAt,
+    createdBy: {
+      user: {
+        id: 1,
+        username: "user1",
+        lastName: userLastName,
+        firstName: userFirstName,
+      },
+    },
+    assignees: [],
+    labels: [],
+    element: undefined,
+  }
```

Comment:
Please move this outside of the test (above) so you can update the object for your second test, instead of duplicating the code on lines 58-70.